src/server/memory/memory.cc

```diff
@@ -420,8 +418,7 @@ Status BulkStoreBase<ID, P>::RemoveOwnership(
     std::set<ID> const& ids, std::map<ID, P>& successed_id_to_size) {
   for (auto id : ids) {
     if (id == EmptyBlobID<ID>() ||
-        id == GenerateBlobID<ID>(reinterpret_cast<void*>(
-                  std::numeric_limits<uintptr_t>::max()))) {
+        id == GenerateBlobID<ID>(std::numeric_limits<uintptr_t>::max())) {
```

Comment:
Use `InvalidObjectID()`.

Reply:
you mean `GenerateBlobID<ID>(InvalidObjectID())` ?

Reply:
sry for my mistake, the correct semantic for `GenerateBlobID<ID>(std::numeric_limits<uintptr_t>::max())` is to generate `0xFFFFFFFFFFFFFFFF` which is `0x8000000000000000UL | 0x0x7FFFFFFFFFFFFFFFUL`.

Now fixed in my 3rd commit, plz check